### PR TITLE
DR-2309 Remove references to solr 3.5 resources

### PR DESCRIPTION
**Pull request by croyfish**
## Jira Tickets ##
[Remove references to solr 3.5 resources](https://jira.nypl.org/browse/DR-2309)

## Things Done ##
- Removed all rels-ext code that was relying on a solr 3.5 core

## How to Test ##
- Unfortunately, we cannot test these changes prior to production(!) due to the fact that the rels_ext connection was not previously possible in QA. The best we can do is follow the testing steps in https://github.com/NYPL/fedora_ingest_rails/pull/97.

### CODE REVIEW

File: app/helpers/ingest_job_helper.rb [hunk 1]
```diff
@@ -136,28 +130,11 @@ def ingest!(ingest_request, test_mode = false)
...
-        if rels_ext.blank?
-          # Remove it from solr as there is a good chance it is there and should not be.
-          rels_ext_index_client.remove_doc_for(uuid)
-        else
-          # Write the doc to solr
-          rels_for_indexing = mms_client.full_rels_ext_solr_docs_for(uuid)
-          rels_indexer_response = rels_ext_index_client.post_solr_doc(uuid, rels_for_indexing)
-
-          # Post the datastream to the repository
-          fedora_client.repository.add_datastream(pid: pid, dsid: 'RELS-EXT', content: rels_ext, mimeType: 'application/rdf+xml', checksumType: 'MD5', dsLabel: 'RELS-EXT XML record for this object')
```
Comment (emu47) [inline app/helpers/ingest_job_helper.rb:150]: I think we can keep this part. This is just posting information to Fedora, which might come in handy later.

File: app/helpers/ingest_job_helper.rb [hunk 2]
```diff
@@ -106,9 +103,6 @@ def ingest!(ingest_request, test_mode = false)
         end
       end
 
-      # Datastreams with info from the `Capture` Level
-      rels_ext = mms_client.rels_ext_for(uuid)
```
Comment (emu47) [inline app/helpers/ingest_job_helper.rb]: I'd retain this for the sake of adding information as a datastream

File: app/helpers/ingest_job_helper.rb (hunk 1 above)
Comment (croyfish) [reply app/helpers/ingest_job_helper.rb:150]: That answers my follow-up question from above!

**Review by chrisb87 — APPROVED** [review]
This is looking right to me. Just going to have to test it live.

**Pull request merged**